bridge/bridge.go

```diff
@@ -216,8 +217,15 @@ func (b *Bridge) add(containerId string, quiet bool) {
 		return
 	}
 
+	// Exposed ports without assigned host ports are ignored, but assigning host ports can take some
+	// time after a container is started. Sleeping gives more time to assign host ports before
+	// registering them
+	log.Println("DEBUG: sleeping")
+	time.Sleep(1 * time.Second)
+
 	servicePorts := make(map[string]ServicePort)
 	for key, port := range ports {
+		log.Println("DEBUG:", container.ID[:12], "ExposedPort", port.ExposedPort, "HostPort", port.HostPort)
```

Comment:
When registrator ignores our service, the `HostPort` string is empty. When it doesn't ignore our service, it's not empty. This is how we figured out it was a race condition.

TODO: If we end up merging this, the added logs should be removed first